script/run-gn-format.py

```diff
@@ -15,6 +15,7 @@ def main():
   for gn_file in sys.argv[1:]:
     subprocess.check_call(
       ['gn', 'format', gn_file],
+      shell=True,
```

Comment:
This doesn't seem like a good fix to me. `shell=True` has [known security issues](https://docs.python.org/2/library/subprocess.html#frequently-used-arguments), and if you end up on a branch that has a gn file called `"| rm -rf /.gn"` you're gonna have a bad time.

Can you explain how this fixes the issue? Can we find a better fix?